src/orquestra/sdk/_ray/_dag.py

```diff
@@ -894,12 +894,8 @@ def stop_workflow_run(self, workflow_run_id: WorkflowRunId) -> None:
         self._client.cancel(workflow_run_id)
 
     def _get_log_reader(self) -> LogReader:
-        if self._service_manager.is_fluentbit_running():
-            return _query_service.FluentbitReader(
-                logs_dir=_services.fluentbit_output_path()
-            )
-        else:
-            return _ray_logs.DirectRayReader(_services.ray_temp_path())
+        # TODO: change to an i-var
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏TODO without ticket